cmd/cluster-agent/subcommands/diagnose/command.go

```diff
@@ -45,15 +39,14 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				}),
 				core.Bundle(),
 				compressionimpl.Module(),
-				diagnosesendermanagerimpl.Module(),
 			)
 		},
 	}
 
 	return []*cobra.Command{cmd}
 }
 
-func run(diagnoseSenderManager diagnosesendermanager.Component, secretResolver secrets.Component) error {
+func run(_ config.Component) error {
```

Comment:
Why keep this argument if it's unused?

Reply:
As some code still use the global `config.Datadog`, we have to make sure `config.Component` is created as it also initializes `config.Datadog`.